etl_framework/BaseEtlSetUp.py

```diff
@@ -183,9 +183,12 @@ def set_etl_job_cutoff_value(self, datetime_cutoff=None):
             pass
 
         else:
-            sql_statement = self.ETL_JOB_SELECT_CUTOFF_STATEMENT%(self.ETL_JOB_ID, )
+                sql_statement = self.ETL_JOB_SELECT_CUTOFF_STATEMENT%(self.ETL_JOB_ID, )
 
-            datetime_cutoff = self.sql_database.run_statement(sql_statement, fetch_data=True)[0][0][0]
+            try:
+                datetime_cutoff = self.sql_database.run_statement(sql_statement, fetch_data=True)[0][0][0]
+            except IndexError as e:
```

Comment:
@hdahme I think I know what IndexError is getting thrown.  It's because we don't have `etl_job` for sfdc-etl set in _etl_jobs_ table.  You could do:

```
except IndexError:
    raise Exception("Datetime Cutoff not set.  Are you sure etl job : {} is set in {}".format(self.ETL_JOB_ID, self.ETL_JOB_TABLE))
```
or something like that